modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -396,14 +400,20 @@ func (h *Handler) doActiveHealthCheck(dialInfo DialInfo, hostAddr string, networ
 		u.Path = h.HealthChecks.Active.Path
 	}
 
+	// if body is provided, create a reader for it, otherwise nil
+	var requestBody io.Reader
+	if h.HealthChecks.Active.Body != "" {
+		requestBody = strings.NewReader(h.HealthChecks.Active.Body)
+	}
+
```

Comment:
Agreed. Though in reality it would only let `{env.*}` and `{file.*}` work, not much else. There's no request context available here in active health checks because they happen in their own goroutines.

Reply:
Oh yeah I did thing about doing that, but then I guess I forgot all about it again :-) 

I'll give it go.

Reply:
OK, I've add a replacer to the body in my latest commit. I've moved the initialization of replacer used for the headers up, and reused that. Is that a good solution ?